sw/host/opentitantool/src/backend/mod.rs

```diff
@@ -15,7 +17,7 @@ pub mod verilator;
 
 #[derive(Debug, StructOpt)]
 pub struct BackendOpts {
-    #[structopt(long, default_value, help = "Name of the debug interface")]
+    #[structopt(long, help = "Name of the debug interface")]
```

Comment:
We may need `default_value` here so that structopt will initialize `interface` to empty string if it is absent from the command line.